server/modules/form/api.js

```diff
@@ -8,17 +8,21 @@ import ES from '../es/service';
 import Marketo from '../marketo/service';
 import * as errors from '../marketo/constants';
 
-import { ERROR_COOKIE } from './constants';
+import { SUBMISSION_COOKIE, ERROR_COOKIE } from './constants';
 
 const logger = new MaskLogger(['email', 'password']);
 
 export default {
 	form: (req, res, next) => {
 
 		let error = req.cookies[ERROR_COOKIE];
-		res.clearCookie(ERROR_COOKIE);
+		if (error) {
+			res.clearCookie(ERROR_COOKIE);
+		}
 
 		res.set('Cache-Control', res.FT_HOUR_CACHE);
```

Comment:
Im pretty sure (just trying to confirm) that we want `cache-control` to be set to `FT_NO_CACHE`, as this will result in browser caching.
Instead we rely on fastly caching via the `surrogate-control` header.